java/runtime/src/main/java/io/ray/runtime/RayNativeRuntime.java

```diff
@@ -96,7 +96,7 @@ public void start() {
     objectStore = new NativeObjectStore(workerContext);
     taskSubmitter = new NativeTaskSubmitter();
 
-    LOGGER.info("RayNativeRuntime started with store {}, raylet {}",
```

Comment:
can we revert this commit 0a743ff5889ae76875a86c0c27c17f1ee67e78d2? java logging is controlled separately in log4j config file. the write way is to change the log threshold in CI, not here.

Reply:
Hmm I'll message you, I'm not sure what the best solution is. Some of the `info` logging is useful but some of it is redundant, and changing it all might not give the best result.